test

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Local validation workflows and try to build documentation locally
+echo "1/5 Running precommit"
+pre-commit run --all-files
+
+echo "2/5 Validating table schemas"
+pre-commit run --all-files
+echo "Processing file: $file"
+for file in spec/*schema.json; do
+  echo "Validating: $file"
+  frictionless validate $file
+done
+
+echo "3/5 Validating datapackage"
+json-schema-cli validate https://specs.frictionlessdata.io/schemas/data-package.json spec/datapackage.json
```

Comment:
This should be `jsonschema-cli`, but even then I get an error. Looks like spec/datapackage.json doesn't exist and should instead be `spec/tides-data-package.json`, but I still get the following error.
```
% jsonschema-cli validate https://specs.frictionlessdata.io/schemas/data-package.json spec/tides-data-package.json
JSON/YAML string must be a a map, cannot load "https://specs.frictionlessdata.io/schemas/data-package.json"
```

Reply:
thx for flagging!  It was too silent a fail for me to notice in my multitasking.  I updated the script to download the profile first (although frustrating that it doesn't have that feature baked in) and made the whole locally testing script/flow more legible so that info-only is suppressed and errors are explicit.